dev/extra-ml-requirements.txt

```diff
@@ -2,17 +2,17 @@
 ## must install in order to use various MLflow Python modules.
 ##
 # Required by mlflow.azureml
-azureml-sdk==1.2.0; python_version >= "3.0"
+azureml-sdk==1.2.0
 # Required by mlflow.keras
 keras==2.3.1
 # Required by mlflow.sklearn
```

Comment:
Could we also consolidate these comments, since they're all kind of repetitive and identical in the form:
`# required by mlflow.x
x
`
i.e. something like "each of these libraries are required by their respective mlflow library APIs." or better wording to that effect?

Reply:
Update: no